src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/AlignedAssemblyOrExcuse.java

```diff
@@ -326,12 +326,23 @@ static void writeAssemblySAMFile(final String outputAssembliesFile,
         final SAMFileHeader cleanHeader = new SAMFileHeader(header.getSequenceDictionary());
         cleanHeader.addReadGroup(contigAlignmentsReadGroup);
         cleanHeader.setSortOrder(assemblyAlnSortOrder);
+        final SAMRecordComparator samRecordComparator;
```

Comment:
This is a trivial suggestion but if you extracted a method to get the comparator you could just have a `return` statement in each `case` clause and wouldn't need the `break`s.

Reply:
done. Method implemented and tested in `SVUtils`.